src/ostorlab/runtimes/lite_local/runtime.py

```diff
@@ -0,0 +1,454 @@
+"""Local runtime runs agents locally.
+
+The local runtime requires Docker Swarm to run robust long-running services with a set of configured services, like
+a local RabbitMQ.
+"""
+import logging
+import socket
+from typing import List
+from typing import Optional
+
+import click
+import docker
+import requests
+import tenacity
+from docker.models import services as docker_models_services
+
+from ostorlab import exceptions
+from ostorlab.assets import asset as base_asset
+from ostorlab.cli import console as cli_console
+from ostorlab.cli import docker_requirements_checker
+from ostorlab.cli import install_agent
+from ostorlab.runtimes import definitions
+from ostorlab.runtimes import runtime
+from ostorlab.runtimes.local import agent_runtime
+from ostorlab.runtimes.local import log_streamer
+from ostorlab.runtimes.local.models import models
+from ostorlab.runtimes.local.services import mq
+
+NETWORK_PREFIX = 'ostorlab_local_network'
+
+logger = logging.getLogger(__name__)
+console = cli_console.Console()
+
+ASSET_INJECTION_AGENT_DEFAULT = 'agent/ostorlab/inject_asset'
+TRACKER_AGENT_DEFAULT = 'agent/ostorlab/tracker'
+LOCAL_PERSIST_VULNZ_AGENT_DEFAULT = 'agent/ostorlab/local_persist_vulnz'
+
+DEFAULT_AGENTS = [
+    ASSET_INJECTION_AGENT_DEFAULT,
+    TRACKER_AGENT_DEFAULT,
+    LOCAL_PERSIST_VULNZ_AGENT_DEFAULT
+]
+
+
+class UnhealthyService(exceptions.OstorlabError):
+    """A service by the runtime is considered unhealthy."""
+
+
+class AgentNotInstalled(exceptions.OstorlabError):
+    """Agent image not installed."""
+
+
+class AgentNotHealthy(exceptions.OstorlabError):
+    """Agent not healthy."""
+
+
+def _has_container_image(agent: definitions.AgentSettings):
+    """Check if container image is available"""
+    return agent.container_image is not None
+
+
+def _is_agent_status_ok(ip: str) -> bool:
```

Comment:
This method is also not used and should be deleted from the other runtime as well.